great_expectations/data_context/store/expectations_store.py

```diff
@@ -60,12 +59,7 @@ def __init__(
             )
 
             # Store Backend Class was loaded successfully; verify that it is of a correct subclass.
-            if issubclass(store_backend_class, TupleStoreBackend):
-                # Provide defaults for this common case
-                store_backend["filepath_suffix"] = store_backend.get(
-                    "filepath_suffix", ".json"
-                )
```

Comment:
The store should not know about its backend's internals. We always want JSON files so let's just update the store backend class